tests/update_reference_data.sh

```diff
@@ -38,7 +38,7 @@ tests[spe1oilgas]="flow spe1 SPE1CASE2_OILGAS spe1_oilgas"
 tests[spe1nowells]="flow spe1 SPE1CASE2_NOWELLS spe1_nowells"
 tests[spe1thermal]="flow spe1 SPE1CASE2_THERMAL spe1_thermal"
 tests[ctaquifer_2d_oilwater]="flow aquifer-oilwater 2D_OW_CTAQUIFER ctaquifer_2d_oilwater"
-tests[fetkovich_2d]= "flow aquifer-fetkovich 2D_FETKOVICHAQUIFER fetkovich_2d"
+tests[fetkovich_2d]="flow aquifer-fetkovich 2D_FETKOVICHAQUIFER fetkovich_2d"
```

Comment:
I could have **sworn** that I stared at that line and yet it did not jump out at me.  Thanks for identifying the issue!